src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukSummaryList.scala.html

```diff
@@ -14,11 +14,46 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.Card
+
 @this()
 
 @(params: SummaryList)
 @import params._
-<dl class="@toClasses("govuk-summary-list", classes)"@toAttributes(attributes)>
+
+@summaryCard(card: Card) = {
+  <div class="@toClasses("govuk-summary-card", card.classes)"@toAttributes(card.attributes)>
+    <div class="govuk-summary-card__title-wrapper">
+      @if(card.title.isDefined) {
+      <h@{card.title.get.headingLevel.getOrElse(2)} class="@toClasses("govuk-summary-card__title", card.title.get.classes)">
+        @card.title.get.content.asHtml
+      </h@{card.title.get.headingLevel.getOrElse(2)}>
+       }
+     @if(card.actions.map(_.items).getOrElse(Seq.empty).nonEmpty){
+       @if(card.actions.map(_.items).getOrElse(Seq.empty).length == 1) {
+         <div class="@toClasses("govuk-summary-card__actions", card.actions.get.classes)">
+         @actionLink(card.actions.get.items.head)
+         </div>
+       } else {
+         <ul class="@toClasses("govuk-summary-card__actions", card.actions.get.classes)">
+         @card.actions.map(_.items).getOrElse(Seq.empty).map { item =>
+           <li class="govuk-summary-card__action">
+             @actionLink(item)
+           </li>
+         }
+         </ul>
+       }
+     }
```

Comment:
similarly something like this (untested)
```suggestion
     @card.actions match {
       case None =>
       case Some(actions) => actions.items match {
         case Nil =>
         case item :: Nil =>
           <div class="@toClasses("govuk-summary-card__actions", actions.classes)">
             @actionLink(item)
           </div>
         case items =>
           <ul class="@toClasses("govuk-summary-card__actions", actions.classes)">
             @items.map { item =>
               <li class="govuk-summary-card__action">
                 @actionLink(item)
               </li>
             }
           </ul>
       }
     }
```

Reply:
@ellamdav I had issues with nested case matching in Twirl - one level of case statements seems fine but the second level was having issues compiling - however I have used one level of case matching to make the code significantly cleaner, be great to get your feedback.